tests/always/test_example_dags.py

```diff
@@ -68,7 +70,7 @@ def example_dags_except_db_exception():
     return [
         dag_file
         for dag_file in example_not_suspended_dags()
-        if any(not dag_file.endswith(e) for e in NO_DB_QUERY_EXCEPTION)
+        if not any(dag_file.endswith(e) for e in NO_DB_QUERY_EXCEPTION)
```

Comment:
Fortunately `NO_DB_QUERY_EXCEPTION` has only one entry now so there’s no functional issues…

Reply:
Correct, while understanding these failing tests, I added `example_redis_publish.py` for a moment to `NO_DB_QUERY_EXCEPTION` and spotted this bug, but for one element it (luckily(?)) works the same.